pkg/asset/machines/aws/machines.go

```diff
@@ -72,10 +77,70 @@ func Machines(clusterID string, region string, subnets map[string]string, pool *
 			},
 		}
 
+		replicas := int32(total / numOfAZs)
+		if int64(idx) < total%numOfAZs {
+			replicas++
+		}
```

Comment:
Replicas for the CPMS should just be total, there's a single CPMS per cluster as opposed to per AZ in MachineSets